src/averageAges.js

```diff
@@ -20,6 +20,13 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = century ? people.filter(
+    (person) => person.sex === 'm'
+    && century === Math.ceil(person.died / 100)) : people.filter(
+    (person) => person.sex === 'm');
```

Comment:
```suggestion
  const men = century
    ? people.filter((person) => (
      person.sex === 'm' && century === Math.ceil(person.died / 100))
    : people.filter((person) => person.sex === 'm');
```
code style - `?` and `:` on new line
+ you can use ternary operator inside filter and avoid duplicating `filter`? bit it is good too